File: ProAgil.Domain/Entities/Event.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace ProAgil.Domain.Entities
{
    public class Event
    {
        public int Id { get; set; }
        [Column(TypeName = "varchar(500)")]
        public string Local { get; set; }
        public DateTime EventDate { get; set; }
        [Column(TypeName = "varchar(500)")]
        public string Theme { get; set; }
        public int PersonQtd { get; set; }
        [Column(TypeName = "varchar(500)")]
        public string ImageURL { get; set; }
        [Column(TypeName = "varchar(500)")]
        public string ContactPhone { get; set; }
        [Column(TypeName = "varchar(500)")]
        public string ContactEmail { get; set; }

        public List<Lot> Lots { get; set; }
        public List<SocialNetWork> SocialNetworks { get; set; }
        public List<SpeakerEvent> SpeakerEvents { get; set; }

    }
}

File: ProAgil-App/src/app/models/Event.ts
import { SpeakerEvent } from './SpeakerEvent';
import { Lot } from './Lot';
import { SocialNetwork } from './SocialNetwork';

export interface Event {
    id: number;
    theme: string;
    local: string;
    eventDate: Date;
    personQtd: number;
    imageURL: string;
    contactPhone: string;
    contactEmail: string;
    lots: Lot[];
    socialNetWorks: SocialNetwork[];
    speakerEvents: SpeakerEvent[];
}
